test/WebSites/BasicWebSite/Startup.cs

```diff
@@ -36,7 +35,7 @@ public void ConfigureServices(IServiceCollection services)
                 var previous = options.InvalidModelStateResponseFactory;
                 options.InvalidModelStateResponseFactory = context =>
                 {
-                    var result = (BadRequestObjectResult) previous(context);
+                    var result = (BadRequestObjectResult)previous(context);
```

Comment:
As above, if anyone objects to this test cleanup in a patch, I'll revert changes to this file.